src/ui/MerchPressModal.jsx

```
@@ -0,0 +1,186 @@
+
+            {/* Actions */}
+            <div className="mt-8 flex justify-end gap-4">
+              <GlitchButton
+              variant="secondary"
+              onClick={onClose}
+              className="uppercase"
+            >
+              [ {t('ui:button.cancel', { defaultValue: 'CANCEL' })} ]
+            </GlitchButton>
+            {!canPress ? (
+              <span tabIndex={0}>
```

Biome (2.4.7)

[error] The HTML element span is non-interactive. Do not use tabIndex.

(lint/a11y/noNoninteractiveTabindex)